ratchet_rs/src/lib.rs

```diff
@@ -44,7 +44,13 @@
     unused_import_braces
 )]
 
-pub use ratchet_core::{self, *};
+pub use ratchet_core::{
+    accept, accept_with, subscribe, subscribe_with, CloseCode, CloseReason, CloseState, Error,
```

Comment:
Is the intention to publish `handshake` and others?

Reply:
@nakedible-p they're published in the core crate and not in the rollup one so you will need to depend on that if you want the functionality on its own: https://github.com/swimos/ratchet/blob/handshake/ratchet_core/src/lib.rs#L80

Reply:
Ah, right, understood.